.gitignore

```diff
@@ -0,0 +1,75 @@
+
```

Comment:
Although I dunno why you would want to remove gitignore as it makes future collaborator's life easier i think